src/EventTracker.java

```diff
@@ -0,0 +1,63 @@
+import components.map.Map;
+
+/**
+ * This class is mean to simulate a single event.
+ */
+public class EventTracker {
+
+    /**
+     * Map that stores all the events.
+     */
+    private Map<String, String> events;
+
+    /**
+     * No argument constructor.
+     */
+    public EventTracker() {
+        this.events = new Map2L<>();
+    }
+
+    //Kernel methods
+    /**
+     * Adds a specific event to the events map.
+     *
+     * @param name
+     *            the name of the event
+     * @param date
+     *            the date of the event
+     */
+    public void addEvent(String name, String date) {
+        events.add(name, date);
+    }
+
+    /**
+     * Removes a specific event from the events map.
+     *
+     * @param name
+     *            the name of the event
+     */
+    public void deleteEvent(String name) {
+        events.remove(name);
+    }
+
+    /**
+     * Adds a category to an event.
+     * @param name the name of the event
+     * @param category the name of the category
+     * @param date the date of the event
+     */
+    public void addCategory(String name, String date, String category) {
```

Comment:
I don't love this method. I would probably just overload the `addEvent` method. One with a category and one without. If you want to add a category later, I might make an `applyCategory` method with maybe the event ID. That way, you don't have to worry about multiple events with the same name and/or date.